D.sublime-build

```diff
@@ -7,31 +7,32 @@
   [
     {
       "name": "Run",
-      "cmd": ["rdmd", "-g", "-debug", "$file"]
+      "working_dir": "$file_path",
+      "cmd": ["rdmd", "-g", "-debug", "$file_name"]
     },
     {
       "name": "dub",
-      "working_dir": "$project_path",
```

Comment:
`$folder` points to the currently open folder. I haven't used Sublime projects before, so sorry if I'm misunderstanding something.